contracts/contracts/utils/InitializableAbstractStrategy.sol

```diff
@@ -68,28 +77,40 @@ abstract contract InitializableAbstractStrategy is Initializable, Governable {
     function _initialize(
         address _platformAddress,
         address _vaultAddress,
-        address _rewardTokenAddress,
+        address[] calldata _rewardTokenAddresses,
         address[] memory _assets,
         address[] memory _pTokens
     ) internal {
         platformAddress = _platformAddress;
         vaultAddress = _vaultAddress;
-        rewardTokenAddress = _rewardTokenAddress;
+        rewardTokenAddresses = _rewardTokenAddresses;
+
         uint256 assetCount = _assets.length;
         require(assetCount == _pTokens.length, "Invalid input arrays");
         for (uint256 i = 0; i < assetCount; i++) {
             _setPTokenAddress(_assets[i], _pTokens[i]);
         }
+        uint256 rewardsCount = _rewardTokenAddresses.length;
```

Comment:
This looks like a local variable. Is it used anywhere?

Reply:
tnx removing